homeassistant/components/devolo_home_network/entity.py

```diff
@@ -21,17 +21,17 @@ def __init__(
         """Initialize a devolo home network device."""
         super().__init__(coordinator)
 
-        self._device = device
-        self._device_name = device_name
+        self.device = device
+        self.device_name = device_name
 
         self._attr_device_info = DeviceInfo(
-            configuration_url=f"http://{self._device.ip}",
-            identifiers={(DOMAIN, str(self._device.serial_number))},
+            configuration_url=f"http://{self.device.ip}",
+            identifiers={(DOMAIN, str(self.device.serial_number))},
             manufacturer="devolo",
-            model=self._device.product,
-            name=self._device_name,
-            sw_version=self._device.firmware_version,
+            model=self.device.product,
+            name=self.device_name,
+            sw_version=self.device.firmware_version,
         )
         self._attr_unique_id = (
-            f"{self._device.serial_number}_{self.entity_description.key}"
+            f"{self.device.serial_number}_{self.entity_description.key}"
         )
```

Comment:
Why drop the `_`?

Reply:
I access device in _is_connected_to_router and with _ pylint complains about the private access.

Reply:
Since this is all in `__init__`, you could simply use `device` and `device_name` function arguments instead of `self.xxx`

Reply:
hmm, I'm not seeing it. Could you help me out? The signatures of _is_connected_to_router and the lambda (and any possible upcoming sensor) would need to be the same although _is_connected_to_router is most likely the only one needing it and having the information available inside the entity, right? Isn't then accessing entity.device (and by that making it public) cleaner?

Reply:
You keep the change at the top (lines 24/25):
```python
        self.device = device
        self.device_name = device_name
```

But in the subsequent lines instead of replacing `self._dev...` by `self.dev...` you replace it by `dev...` (drop the `self.`)

Reply:
Got it! I was at a complete different spot in the code ...